naas/models/account.py

```diff
@@ -0,0 +1,49 @@
+from naas.models import Links, AccountSetting
+
+
+class Account(object):
+    """
+
+    Account
+    ===============
+
+    This returns an instance of the account domain model
+    """
+
+    def __init__(self, attributes={}):
+        self.attributes = attributes
+
+    def id(self):
+        """Returns the subscriber id"""
+        return self.attributes.get('id')
+
+    def name(self):
+        """Returns the account name"""
+        return self.attributes.get('name')
+
+    def account_setting_attributes(self):
+        """Returns the attributes for the account setting"""
+        return self.attributes.get('account_setting', {})
+
+    def account_setting(self):
+        """Returns the account setting"""
+        if self.account_setting_attributes():
+            return AccountSetting(self.account_setting_attributes())
+        else:
+            return AccountSettings.retrieve()
```

Comment:
This revealed a previous error for `AccountSettings`. This delegates to the `AccountSettings` model, which has a static method trying to make a request to `AccountSettings.retrieve()` (But there is no request). Just making note of that here.